com/win32com/client/build.py

```diff
@@ -382,6 +382,22 @@ def MakeDispatchFuncMethod(self, entry, name, bMakeClass=1):
         if doc and doc[1]:
             ret.append(linePrefix + "\t" + _makeDocString(doc[1]))
 
+        for i in range(len(fdesc[2])):
+            desc = fdesc[2][i]
+
+            if (
+                desc[1] & (pythoncom.PARAMFLAG_FOUT | pythoncom.PARAMFLAG_FIN)
+                == pythoncom.PARAMFLAG_FOUT
+            ):
+                if (
+                    desc[0] & pythoncom.VT_RECORD
+                    and desc[3]
+                    and desc[3].__class__.__name__ == "PyIID"
+                ):
+                    newVal = f"pythoncom.GetRecordFromGuids(CLSID, MajorVersion, MinorVersion, LCID, {repr(desc[3])})"
+                    ret.append(linePrefix + "\t" + f"if {names[i+1]} == {defOutArg}:")
+                    ret.append(linePrefix + "\t\t" + f"{names[i+1]} = {newVal}")
```

Comment:
Small suggestion, since you're already using an f-string
```suggestion
                    ret.append(f"{linePrefix}\tif {names[i+1]} == {defOutArg}:")
                    ret.append(f"{linePrefix}\t\t{names[i+1]} = {newVal}")
```